### PR TITLE
Add support for symfony 6.*

### PR DESCRIPTION
**Pull request by lebris**
- add support for symfony `6.x`
- bump `phpunit` version to `9.5`
- bump minimal php version to `8.0`
- migrate from travis to github actions
- codecov migrated as-is -> require more time to update and setup (seems broken for a while)

=> Requires a new major release `5.0`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,6 +207,10 @@ $verbosity = $config->readRequired('verbosity');
 Changelog
 ---------
 
+**4.x --> 5.x**
+
+ - Drop php =< 7.2 support. Minimal version is 7.3.0
```
Comment (Niktux) [inline README.md:212]: 8.0 ?
cf composer.json

**Review by Niktux — APPROVED** [review]
GTM

File: README.md (hunk 1 above)
Comment (lebris) [reply README.md]: :+1:

Comment (lebris) [reply README.md]: 22fc46f

**Pull request merged**